v3/src/components/graph/utilities/pixi-points.ts

```diff
@@ -454,10 +457,17 @@ export class PixiPoints {
       if (point.texture !== texture) {
         point.texture = texture
         const metadata = this.getMetadata(point)
-        metadata.style = style
+        metadata.style = newStyle
       }
     }
 
+    // If we're animating the change, we need to transition the point scale.
+    animateChange && this.points.forEach((point) => {
+      this.transition(() => {
+        this.setPointScale(point, style.radius / currentRadius)
+      }, { duration: transitionDuration })
+    })
```

Comment:
Would it be possible to call these lines outside of this function? It seems strange to me that `animateChange` triggers the animation of only one specific style property (point radius). Seeing a call to `matchPointsToData` with the `animateChange` parameter set to true, I would expect that all updated style properties would be animated.